src/web/views/locales/cy/confirm.html

```diff
@@ -5,7 +5,7 @@ <h2 class="govuk-heading-m">Pulvinar placerat odio</h2>
     <li>Sed nisl felis, commodo eu bibendum.</li>
     <p class="govuk-body">In hac habitasse platea dictumst donec in purus et enim sodales pharetra.</p>
     <li>Quisque vel ex porttitor accumsan massa egestas.</li>
-    <p class="govuk-body">Maecenas id molestie orci nec suscipit neque vivamus 4 ornare dui libero a finibus libero faucibus at duis tristique. Morbi porta tellus £12.40 diam 4 luctus volutpat lectus scelerisque pellentesque proin.</p>
+    <p class="govuk-body">Maecenas id molestie orci nec suscipit neque vivamus 4 ornare dui libero a finibus libero faucibus at duis tristique. Morbi porta tellus £<span data-entitlement="total-voucher-value-four-weeks">{{ totalVoucherValueForFourWeeks }}</span> diam 4 luctus volutpat lectus scelerisque pellentesque proin.</p>
```

Comment:
`<span data-entitlement="total-voucher-value-four-weeks">` has been added as a hook for acceptance tests. I haven't added an acceptance test in this PR to keep it focused. I will create another PR that will allow for asserting on possible multiple instances of voucher entitlement values.